### DISCUSSION TITLE
0.5.2

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#986] [Announcements]
- Improve library design
- Lyrics translator (full version only)
- Minor enhancement and bug fixes

[Compare versions](https://github.com/z-huang/InnerTune/wiki/App-Versions)

Note: Full version has larger apk size because the translation library is bundled inside.

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.2'>0.5.2</a>.</em>

**Comment by ANDROPRROS** [comment]
I tried the new version, everything seems to be fine, but the albums do not load, this happens when entering an artist and want to see all the albums, use the version arm64

**Comment by stepersy** [comment]
How does Translation work? If I am Italian, how can I choose the Italian Translation of the lyrics?

**Reply by z-huang** [reply]
The translator translates the lyrics to the language of your phone. If your phone's language is Italian, it should translate the lyrics to Italian.

**Reply by stepersy** [reply]
I have the phone settled in English. There isn't any setting in the app that allows me to change the App language? Thanks

**Comment by matt3249** [comment]
One of the best music apps ever, way to go mate!!

**Comment by kasra996** [comment]
Hi, there is many installation files and note here, so I'm a bit confused, what is each installation file for?? Which should i install??

**Reply by ANDROPRROS** [reply]
Foss is the universal version, you can install that without problems, the versions that say full, just install the version that corresponds to your processor, arm64 for Snapdragon, armeabi-v7a for MediaTek, Look for the model of your phone to find out what architecture it is, if you want the version with a letter translator

**Comment by marcossmtp** [comment]
In previous version when you visualize a album you had the button to add to library that album.
Now the only option is open a song from the album and in the banner menu of the song is the button to add the album.
Is not more logical has the button in the  album visualitation?
Or at least in both of them

**Reply by z-huang** [reply]
"Add to library" is renamed to "like" for albums, while library albums are albums of your library songs. Does this answer your question?

**Reply by marcossmtp** [reply]
Yes, now its clear. thanks

**Comment by Fusillo3D** [comment]
Hello, in this version there's no more the ability to swipe on the album image to change the music.

Also, please, could you make the bottom bar (Home, Tracks, Albums) always visible or at least, place something on the top, maybe small icons or just a Home icon? Because when you're searching for something, the only way to go to different sections (Albums etc), is go back multiple times until you're in the Home.
Something like the example in my screenshot.
![Screenshot_20230831_215913_InnerTune](https://github.com/z-huang/InnerTune/assets/30672630/e83293b4-aee8-4270-a78b-df76b9b316f0)

Thanks for the attention, I think this is the perfect client overall. Appreciate your work!
